driver/syscall_table.c

```diff
@@ -373,6 +373,9 @@ const struct syscall_evt_pair g_syscall_table[SYSCALL_TABLE_SIZE] = {
 #ifdef __NR_copy_file_range
 	[__NR_copy_file_range - SYSCALL_TABLE_ID0] =            {UF_USED, PPME_SYSCALL_COPY_FILE_RANGE_E, PPME_SYSCALL_COPY_FILE_RANGE_X},
 #endif
+#ifdef __NR_capset
```

Comment:
Is there no `__NR_ia32_capset` or did you forget about it? :D

Reply:
The second one, of course 😄